### ISSUE TITLE
As a content editor, I want an easy way to get from an account record to the associated person record so I can view and correct person details.

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/mep-django#56]
## Notes for testing
On the account edit page, on the account holder section, you should now see an edit link for each person associated.  Should work when loading an existing record with a person associated and also immediately after you add a new person.  Edit links should be displayed in the same order as the people and links should open a new window or tab.

**Comment by rlskoeser** [comment]
Estimated as a 1 - if it's significantly harder we should reconsider.

**Comment by elspethgreen** [comment]
This actually may be pretty important; or rather, an analogous situation involving subscriptions/renewals and accounts.
The association mechanism works in the same way: you type in the beginning of the account name, and it gives you options.  In the event that one (or more) names appears, it would be really useful to be able to navigate easily to the accounts themselves to check their records of subscription events.  
I just did this with "Buckminster"--I was pretty sure he was unique, but I checked by seeing whether Buckminster had any activity at the library around the time of the subscription event I was adding.  Indeed he had!  But I had to back all the way out of the subscription-add feature to do it.

**Comment by rlskoeser** [comment]
@elspethgreen  I think this issue is no longer needed, now that we've come up with a different approach to display the person information inline on the account edit page.  Or should we still look for a way to provide a link to the full person record?

**Comment by elspethgreen** [comment]
Agreed--as long as we can see enough person info on the account page, we don't need this linking functionality.

**Comment by elspethgreen** [comment]
A question: I remember discussing the possibility of associating multiple "persons" with a given "account" if desired--that there would be, as with the "subscription" addition section, an "add another person" button.  I realize now that this isn't actually reflected in any of these admin stories.  It's not completely urgent, as it won't happen a lot, but it would be useful!

**Comment by rlskoeser** [comment]
@elspethgreen I think the behavior you want will be included in #63.  Should we add a note there to make sure you remember to test this?

**Comment by elspethgreen** [comment]
Yes!  Sorry, I'm not sure how I missed that.  I don't think a note is needed--once it comes up for testing, it should be clear.

**Comment by clmahoney** [comment]
Is this what we're talking about? 
![screen shot 2018-02-17 at 6 30 58 pm](https://user-images.githubusercontent.com/16673934/36346671-e89ff80c-1410-11e8-997b-0f6d0cee4f74.png)

Also I noticed something odd: in the MEP 2 site, Buckmaster is Account #4260, but on the Test Side Buckmaster is nowhere to be found and de Forceville is Account #4260. Why is this??

**Comment by rlskoeser** [comment]
@clmahoney Sorry, I didn't add any testing notes!  On the account edit page at the top, in the account holder section you should see an 'edit' link next to the autocomplete box.  If there are multiple people associated, you should see multiple edit links displayed in the same order.

**Comment by rlskoeser** [comment]
@clmahoney the discrepancy in accounts is likely because we haven't refreshed the test site with data from production in a while.  Is Buckmaster someone you added with the new logbook data you entered?

**Comment by elspethgreen** [comment]
Buckmaster is indeed someone added from the logbook entries!

**Comment by elspethgreen** [comment]
I took a look at this, and it seems to be working great.  I tried editing Elvira de Alvear by moving to her personography record from her account record.  I added the title "Madame," saved the edit, then refreshed the account page--et voila!  "Madame Elvira de Alvear."
I also checked out people with the same account name (multiple "Andrews," for example) and the edit feature works exactly as I hoped (as does the list of subscription events in the account info).
